CDS/src/org/icpc/tools/cds/ConfiguredContest.java

```diff
@@ -747,9 +747,20 @@ protected IContestObject filterHidden(IContestObject obj) {
 			ITeam team = contest.getTeamById(clar.getFromTeamId());
 			if (contest.isTeamHidden(team))
 				return null;
-			team = contest.getTeamById(clar.getToTeamId());
-			if (contest.isTeamHidden(team))
-				return null;
+			if (clar.getToTeamIds() != null) {
```

Comment:
I wonder if it would be better to only hide the clarification if **all** teams and groups in it are hidden and if not, only hide the team / group IDs of hidden ones? It does complicate stuff though.